### DISCUSSION TITLE
kindling-collecotr在安装的时候，出现机器一分钟负载飙高300+的情况，后续无法复现

### DISCUSSION THREAD

**Discussion by zcl1115** [harmonycloud/kindling#583] [agent collector]
在一次kindling-collector的安装过程中，出现了机器的一分钟负载飙高到300+的情况，内核日志显示如下，但是看不出有异常情况，后续也没有办法复现
![企业微信截图_16966500704585](https://github.com/KindlingProject/kindling/assets/45005291/ae0e1910-3904-4ccd-bd04-43cd72981200)

**Comment by dxsup** [comment]
日志显示当时正在为每个CPU创建buffer，每个大小为8MB，如果主机CPU核数较大，可能会出现短暂的负载升高

**Reply by zcl1115** [reply]
这是执行机器的信息，应该是只有40个核的，但是这个内核日志是申请到70+？
root@yarn2-slave5:/# cat /proc/cpuinfo| grep "physical id"| sort| uniq| wc -l
2
root@yarn2-slave5:/# cat /proc/cpuinfo| grep "cpu cores"| uniq
cpu cores       : 20
root@yarn2-slave5:/# uname -a
Linux yarn2-slave5 4.9.70-040970-generic #201712161132 SMP Sat Dec 16 16:33:52 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
root@yarn2-slave5:/# cat /proc/version
Linux version 4.9.70-040970-generic (kernel@gloin) (gcc version 6.3.0 20170406 (Ubuntu 6.3.0-12ubuntu2) ) #201712161132 SMP Sat Dec 16 16:33:52 UTC 2017
